content/src/fundings/angel-match.md

```diff
@@ -1,5 +1,5 @@
 ---
-filename: Angel Match Program
+filename: angel-match
```

Comment:
[sand]: might be good to make this field write-once-read-only if its being editable which is seems like it is.

Reply:
It already is!